src/frontend/src/lib/components/tokens/TokenLogo.svelte

```diff
@@ -49,5 +51,14 @@
 				testId={`network-${badgeTestId}`}
 			/>
 		</div>
+	{:else if badge?.type === 'transaction'}
+		<!-- TODO: use new mapping color when merged-->
+		<div
+			class="absolute -bottom-1 -right-1 rounded-full bg-brand-tertiary p-1 text-white"
```

Comment:
You need `text-white`?

Reply:
Should we set same value as network - i.e. `flex h-6 w-6 items-center justify-center`?

Reply:
agreed! done